### ISSUE TITLE
Maximum possible score?

### ISSUE THREAD

**Issue by DuoX371** [l3lackShark/gosumemory#127]
There is max possible PP so I'm wondering if it is possible to get max possible score

**Comment by l3lackShark** [comment]
Int32 limit which then goes down (stable logic handling integer overflow)
~~2B in score

**Comment by DuoX371** [comment]
Sorry let me rephrase it. I meant max possible score in current play/if FC the rest of the map.

**Comment by DuoX371** [comment]
- mainly think it would work out well for tourney client
- possibility of beating current best score on map
- think it would look good for streaming as well
